Sprint-1/implement/dedupe.test.js

```diff
@@ -25,3 +25,35 @@ test.todo("given an empty array, it returns an empty array");
 // Given an array with strings or numbers
 // When passed to the dedupe function
 // Then it should remove the duplicate values, preserving the first occurence of each element
+describe("calculateMedian", () => {
+  test("removes duplicates from an array with duplicates", () => {
+    expect(dedupe([1, 2, 2, 3, 4, 4, 5])).toEqual([1, 2, 3, 4, 5]);
+  });
+
+  test("returns the same array when all elements are unique", () => {
+    expect(dedupe([1, 2, 3, 4, 5])).toEqual([1, 2, 3, 4, 5]);
+  });
+  test("returns the same array when all elements are unique", () => {
```

Comment:
Description does not match.

Also, you can test arrays with duplicate values that appear in different order or scattered at random locations in the array.

Prepare the tests as if you have no knowledge about how the function is implemented, and with the goal to break the function.